test/system/012-manifest.bats

```diff
@@ -20,4 +22,36 @@ load helpers
     run_podman rmi test:1.0
 }
 
+@test "podman manifest --tls-verify and --authfile" {
+    skip_if_remote "running a local registry doesn't work with podman-remote"
+    start_registry
+    authfile=${PODMAN_LOGIN_WORKDIR}/auth-$(random_string 10).json
+    run_podman login --tls-verify=false \
+               --username ${PODMAN_LOGIN_USER} \
+               --password-stdin \
+               --authfile=$authfile \
+               localhost:${PODMAN_LOGIN_REGISTRY_PORT} <<<"${PODMAN_LOGIN_PASS}"
+    is "$output" "Login Succeeded!" "output from podman login"
+
+    manifest1="localhost:${PODMAN_LOGIN_REGISTRY_PORT}/test:1.0"
+    run_podman manifest create $manifest1
+    mid=$output
+    run_podman manifest push --authfile=$authfile \
+        --tls-verify=false $mid \
+        $manifest1
+    run_podman manifest rm $manifest1
```

Comment:
Tests are a little duplicatey, and a little hard to read because theyreallglommedtogetherwithnowhitespace. Not a blocker, but if you need to re-push for something else, how about:
```bash
    ...
    run_podman manifest rm $manifest1

    # Default is to require TLS; also test explicit opts
    for opt in '' '--insecure=false' '--tls-verify=true' "--authfile=$authfile"; do
        run_podman 125 manifest inspect $opt $manifest1
        assert "$output" =~ "Error: reading image \"docker://$manifest1\": pinging container registry localhost:${PODMAN_LOGIN_REGISTRY_PORT}:.*x509" \
               "TLE check: fails (as expected) with ${opt:-default}"
    done

    # Now try with --authfile and with envariable, and both --tls-verify & --insecure
    expect_good="\"mediaType\": \"application/vnd.docker.distribution.manifest.list.v2\+json\""
    run_podman manifest inspect --authfile=$authfile --tls-verify=false $manifest1
    assert "$output" =~ "$expect_good" "--authfile + --tls-verify=false"

    run_podman manifest inspect --authfile=$authfile --insecure $manifest1
    assert "$output" =~ "$expect_good" "--authfile + --insecure"

    REGISTRY_AUTH_FILE=$authfile run_podman manifest inspect --tls-verify=false $manifest1
    assert "$output" =~ "$expect_good" "Works with REGISTRY_AUTH_FILE"
```
Note the addition of `.*x509` to the tls test; IIRC go-something changed error strings, but both old and new included `x509`? I don't know how to easily check that. OK to remove; I just don't think "pinging container registry" is a sufficient test.